.github/buildomat/jobs/package.sh

```diff
@@ -29,6 +29,9 @@ set -o xtrace
 cargo --version
 rustc --version
 
+WORK=/work
+pfexec mkdir -p $WORK && pfexec chown $USER $WORK
```

Comment:
Changing `/work` to `$WORK` makes it easier to run this job by hand on a build machine.  The `mkdir` is moved up to the top because it gets used just below.  I assume this hasn't bitten us before because the directory is created for us in CI.